### PR TITLE
Travis CI Packagecloud integration

### PR DESCRIPTION
**Pull request by zultron**
This is @kinsamanka's Travis CI Packagecloud integration.

If the Travis CI repo settings contain the `PACKAGECLOUD_USER`, `PACKAGECLOUD_REPO` and (private) `PACKAGECLOUD_TOKEN` variables, then Travis CI will upload packages to the packagecloud.io repository after a merge to master and successful package build.

The parent commit is a no-op variable cleanup commit.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@ language: bash
 services:
 - docker
 
+before_install:
+  - gem install package_cloud
+  - pip install --user travispy
```
Comment (bobvanderlinden) [inline .travis.yml]: Where is this used?

Comment (zultron) [inline .travis.yml]: @bobvanderlinden `package_cloud` is used [below][1] in `.travis/docker_run.sh`.  I assume `travispy` is a dependency, but not sure.

[1]: https://github.com/machinekit/machinekit/pull/819/files#diff-a6816f47ee9ac2d3995237c887637d24R82

**Comment by zultron** [comment]
I also snuck in a commit that fixes the timestamp issue.

File: .travis.yml (hunk 1 above)
Comment (kinsamanka) [inline .travis.yml]: line 21 is not used, this was used to synchronise the runtest task with the builders.

**Comment by zultron** [comment]
@kinsamanka @bobvanderlinden Fixed up the patch [here][1]

[1]: https://github.com/zultron/machinekit/commit/eadf006b0755f26e1bf2973f6e40545d712edee2#diff-354f30a63fb0907d4ad57269548329e3R19

**Pull request merged**